core/utils_test.py

```diff
@@ -750,3 +750,21 @@ def test_convert_png_data_url_to_binary_raises_if_prefix_is_missing(
 
     def test_quoted_string(self) -> None:
         self.assertEqual(utils.quoted('a"b\'c'), '"a\\"b\'c"')
+
+    def test_is_base64_encoded(self) -> None:
+        image = '<svg><path d="%s" /></svg>' % (
+            'M150 0 L75 200 L225 200 Z ' * 1000)
+        with python_utils.open_file( # type: ignore[no-untyped-call]
+            os.path.join(feconf.TESTS_DATA_DIR, 'img.png'),
+            'rb', encoding=None
+        ) as f:
+            raw_image = f.read()
```

Comment:
I don't think you need to load the file it doesn't provide any benefit for this test.

Reply:
Done.